File: app/webview.rb
class CustomWebView < WebView
  def keyDown(event)
    # ESC キーとか、キーボードショートカットに対応していないキーが押されるとアラート音がしてやかましいので、
    # FirstResponder をセットして対応。

    # NSLog event.description
  end
end

File: kotori/WebView.swift
import WebKit

class CustomWebView : WebView {
    override func keyDown(with event: NSEvent) {
        // ESC キーとか、キーボードショートカットに対応していないキーが押されるとアラート音がしてやかましいので
        // keyDown を無効にする
    }
}
